GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListViewModel.swift

```diff
@@ -8,12 +8,45 @@
 
 import SwiftUI
 import Foundation
+import Combine
 
 class DogBreedsListViewModel: ObservableObject {
-    @Published var dogBreeds = [
-        DogBreed(name: "nagazi",description: "a",minLife: 4, maxLife: 10,maleMinWeight: 12,maleMaxWeight: 14,femaleMinWeight: 23,femaleMaxWeight: 26),
-                 
-        DogBreed(name: "bulldog",description: "b",minLife: 4, maxLife: 10,maleMinWeight: 12,maleMaxWeight: 14,femaleMinWeight: 23,femaleMaxWeight: 26),
-        DogBreed(name: "labrador",description: "c",minLife: 4, maxLife: 10,maleMinWeight: 12,maleMaxWeight: 14,femaleMinWeight: 23,femaleMaxWeight: 26)
-    ]
+    private let dogBreedsService: DogBreedsServing
+    
+    @Published var dogBreeds = [DogBreed]()
+    
+//    @Published private(set) var hasError: Bool = false
```

Comment:
No need to comment these, just delete them

Reply:
I forgot to delete :(